allennlp/service/cli/run.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+from contextlib import ExitStack
+import json
+import sys
+from typing import Sequence, Optional, IO
+
+from allennlp.service.servable import Servable, ServableCollection
+
+
+def parse_args(args: Sequence[str]):
+    parser = argparse.ArgumentParser(description="Run a model")  # pylint: disable=invalid-name
+    parser.add_argument('model', type=str, help='the name of the model to run')
+    parser.add_argument('input_file', metavar='input-file', type=str, help='path to input file')
+    parser.add_argument('--output-file', type=str, help='path to output file')
+    parser.add_argument('--print', action='store_true', help='print results to string')
+
+    return parser.parse_args(args)
+
+def get_model(args: argparse.Namespace) -> Optional[Servable]:
+    # TODO(joelgrus): use the args to instantiate the model
+    models = ServableCollection.default()
+    model_name = args.model
+    return models.get(model_name)
+
+def run(servable: Servable, input_file: IO, output_file: Optional[IO], print_to_console: bool) -> None:
+    for line in input_file:
+        data = json.loads(line)
+        result = servable.predict_json(data)
+        output = json.dumps(result)
+
+        if print_to_console:
+            print(output)
+        if output_file:
+            output_file.write(output + "\n")
+
+def main(argv: Sequence[str]):
+    args = parse_args(argv)
+    model = get_model(args)
+    if model is None:
+        print("unknown model:", args.model)
+        sys.exit(-1)
+
+    # ExitStack allows us to conditionally context-manage `output_file`, which may or may not exist
+    with ExitStack() as stack:
+        input_file = stack.enter_context(open(args.input_file, 'r'))  # type: ignore
+        if args.output_file:
+            output_file = stack.enter_context(open(args.output_file, 'w'))  # type: ignore
+        else:
+            output_file = None
```

Comment:
I think you missed my point, but if `print(s, file=output_file)` is unidiomatic, then my point doesn't make sense.

```
        if args.output_file:
            output_file = stack.enter_context(open(args.output_file, 'w'))  # type: ignore
       else:
            output_file = sys.stdout
```